README.md

```diff
@@ -76,6 +76,9 @@ const schema = new Schema('album', {
   title: { type: 'text' },
   year: { type: 'number' }
 })
+
+// redis is the connection (client) instance from createClient()
```

Comment:
I don't like the comment. I mean it reads fine, I just feel it's the wrong place to have it. I know it's not clear here how to get a Node Redis connection from this example. But this code isn't _really_ an example. This bit of the document is intended to entice the developer to try Redis OM. The rest of the README is the details on doing that.